browser/brave_browser_main_parts_android.cc

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_browser_main_parts_android.h"
+#include "brave/browser/browsing_data/brave_clear_browsing_data.h"
+
+void BraveBrowserMainPartsAndroid::PreShutdown() {
```

Comment:
This change will affect Android, right? cc: @SergeyZhukovsky @AlexeyBarabash

Reply:
I don't think that will be ever triggered on Android. It is a different process on mobile. Usually system kills an app or user swipes it. 
On Android the approach like this clear on exit can not be used. We will go a different way, clear on start. Any app on Android could be killed at any moment by OS when there are no resources.

Reply:
Thanks @SergeyZhukovsky, I'll remove the sub-classing for Android.